integration/hive/build.gradle

```diff
@@ -0,0 +1,229 @@
+                    }
+                }
+                developers {
+                    developer {
+                        id = "openlineage"
+                        name = "OpenLineage Project"
+                    }
+                }
+                scm {
+                    connection = "scm:git:git://github.com/OpenLineage/OpenLineage.git"
+                    developerConnection = "scm:git:ssh://github.com:OpenLineage/OpenLineage.git"
+                    url = "https://github.com/OpenLineage/OpenLineage"
+                }
+            }
+
+            pom.withXml {
+                asNode().dependencies."*".findAll() {
+                    it.groupId.text() == "hive"
+                }.each() {
+                    it.parent().remove(it)
+                }
+            }
+        }
+    }
+
+    processResources {
+        filter ReplaceTokens, tokens: [
+                "version": project.property("version")
+        ]
+    }
+}
+
+
+nexusPublishing {
+    repositories {
+        sonatype {
+            username = System.getenv('RELEASE_USERNAME')
+            password = System.getenv('RELEASE_PASSWORD')
+        }
+    }
+}
+
+signing {
+    required { isReleaseVersion }
+    def signingKey = findProperty("signingKey")
+    def signingPassword = findProperty("signingPassword")
+    useInMemoryPgpKeys(signingKey, signingPassword)
+    sign publishing.publications.mavenJava
+}
+
+shadowJar {
+    archiveClassifier = ""
+
+    minimize() {
+        exclude(project(path: ":hive-openlineage-hook"))
+    }
+
+    dependencies {
+        exclude(dependency("org.slf4j::"))
+        exclude("org/apache/commons/logging/**")
+        exclude("io/openlineage/server/**")
+        exclude("javax/**")
+        exclude("com/google/**")
+        exclude("org/checkerframework/**")
+        exclude("org/publicsuffix/**")
+        exclude("org/slf4j/**")
+    }
+
+    relocate "com.github.ok2c.hc5", "io.openlineage.hive.shaded.com.github.ok2c.hc5"
+    relocate "org.apache.httpcomponents.client5", "io.openlineage.hive.shaded.org.apache.httpcomponents.client5"
+    relocate "javassist", "io.openlineage.hive.shaded.javassist"
+    relocate "org.apache.hc", "io.openlineage.hive.shaded.org.apache.hc"
+    relocate "org.apache.http", "io.openlineage.hive.shaded.org.apache.http"
+    relocate "org.apache.commons.codec", "io.openlineage.hive.shaded.org.apache.commons.codec"
+    relocate "org.apache.commons.lang3", "io.openlineage.hive.shaded.org.apache.commons.lang3"
+    relocate "org.apache.commons.beanutils", "io.openlineage.hive.shaded.org.apache.commons.beanutils"
+    relocate 'org.yaml.snakeyaml', 'io.openlineage.hive.shaded.org.yaml.snakeyaml'
+    relocate "com.fasterxml.jackson", "io.openlineage.hive.shaded.com.fasterxml.jackson"
+    relocate "org.LatencyUtils", "io.openlineage.hive.shaded.org.latencyutils"
+    relocate "org.HdrHistogram", "io.openlineage.hive.shaded.org.hdrhistogram"
```

Comment:
Have you doublechecked generated jar does not contain any other unshaded dependencies?